### PR TITLE
fix: issue with storing false values in data model not validating as …

### LINKED ISSUE #1746: Required boolean fields do not validate when value set to false
### Description of the bug

When setting a Radio button with a simpleBinding to a boolean field to the value `false`. Validation will fail if the RadioButton has specified `"required": true`. 

The data model returned from the save operation seems to have the correct value set on the field. `funeral.isRelativesInformed` in the screenshot below. 

### Steps To Reproduce

1. Create a boolean field in the datamodel
2. Create a radio button with two options which map to `false` and `true`
3. Select the option that maps to `false`
4. Validate. 

### Additional Information

image
image

### PR DESCRIPTION
**Pull request by mikaelrss**
## Description
This PR fixes an issue where setting falsy values in input fields with boolean or integer data model fields triggered warnings of not being set when submitting or navigating the form. 

## Related Issue(s)

- closes #1746

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
  - [ ] Creator of the original issue (or service owner) has been contacted for manual testing (or will be contacted when released in alpha)
  - [ ] No testing done/necessary
- Automated tests
  - [ ] Unit test(s) have been added/updated
  - [x] Cypress E2E test(s) have been added/updated
  - [ ] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [ ] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [x] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [x] No functionality has been changed/added, so no documentation is needed
  - [ ] I will do that later/have created an issue
  
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [x] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [x] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [x] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

**Comment by mikaelrss** [comment]
The group-test that fails is not related to the functionality of this PR. All relevant tests are passing :innocent:

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "core-js": "^3.29.1",
     "cross-env": "7.0.3",
     "css-loader": "6.8.1",
-    "cypress": "13.6.0",
+    "cypress": "^13.6.3",
```
Comment (olemartinorg) [inline package.json:82]: I suspect [renovate will revert that change](https://github.com/renovatebot/renovate/blob/main/docs/usage/dependency-pinning.md) in the next renovate PR.

**Review by olemartinorg — APPROVED** [review]

**Review by Magnusrm — APPROVED** [review]
Looks good! 🚀

File: package.json (hunk 1 above)
Comment (mikaelrss) [reply package.json:82]: Alright. I'll update it with a pinned version :smile:

**Pull request merged**